test/libsolidity/semanticTests/storageLayoutSpecifier/variable_cleanup sstore.sol

```diff
@@ -0,0 +1,15 @@
+contract C {
+    uint8 x;
+    int8 y;
+    bytes2 b;
+    function f(uint _x, int _y, bytes3 _b) public returns (uint16, int32, bytes2) {
```

Comment:
This could use a comment. At first I thought that the results were wrong, because the returned types actually weren't small enough to truncate the input like this. Only then I noticed that the storage variable types do not match and are actually event smaller.

Reply:
Added a comment explaining the types are larger.